packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -563,7 +570,7 @@ export default class TableResize implements EditorPlugin {
     }
 
     private frameAnimateResizeCells = (e: MouseEvent) => {
-        this.editor.runAsync(() => this.resizeCells(e));
+        this.editor?.runAsync(() => this.resizeCells(e));
```

Comment:
Removed. it was null during the testing since the test code removed the editor but it is not the case in production. Since we are directly testing resizeCells without triggering code here, no need to check null here.